### PR TITLE
Page: Introducing JSX

### PR DESCRIPTION
**Pull request by TheoryOfNekomata**
Add translations for Introducing JSX.

Note: Succeeding pages should also be updated given enough translations to those pages.

### CODE REVIEW

File: content/docs/introducing-jsx.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
 
-Consider this variable declaration:
+Mangyaring tingnan ang variable declaration na ito:
 
 ```js
 const element = <h1>Hello, world!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+Ang syntax na ito ay hindi isang string o HTML.
```
Comment (warrenca) [inline content/docs/introducing-jsx.md:15]: `Para bang hindi matanto kung ito ba ay string o HTML.` - suggestion

Comment (TheoryOfNekomata) [reply content/docs/introducing-jsx.md:15]: I put it that way to reflect the straightforward tone of the English documentation. I think we should keep it, unless the others have a say.

File: content/docs/introducing-jsx.md [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
 
-It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with the full power of JavaScript.
+Ito ay tinatawag na JSX, at isa itong halimbawa ng syntax extension sa JavaScript. Inirerekomenda naming gamitin ito kasama ng React para bigyan ng porma ang UI. Maihahalintulad ang JSX sa isang template language, ngunit maaari rin itong ipaghalo sa iba pang JavaScript na code.
```
Comment (warrenca) [inline content/docs/introducing-jsx.md:17]: - Inirerekomenda = Ipinapayo
- JSX may remind you of a template language = `Ang JSX ay magpapaalala sa inyo ng template language`
- but it comes with the full power of JavaScript = `pero dala nito ang isang daan porsyentong lakas ng JavaScript.` or `pero dala nito ang buong bisa at lakas ng JavaScript.`

File: content/docs/introducing-jsx.md [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
 
-JSX produces React "elements". We will explore rendering them to the DOM in the [next section](/docs/rendering-elements.html). Below, you can find the basics of JSX necessary to get you started.
+Mula sa JSX, makabubuo tayo ng mga React "element". Ipapakita sa [susunod na paksa](/docs/rendering-elements.html) ang pag-render ng JSX papunta sa DOM. Samantala, nasa sumusunod na bahagi ang mga halimbawa ng JSX para masimulan ang paggamit nito.
```
Comment (warrenca) [inline content/docs/introducing-jsx.md:19]: section = sa approved PR https://github.com/reactjs/tl.reactjs.org/pull/3/files ginamit yung word na kabanata. Pwede papalit ng `paksa` to `kabanata` para consistent.

File: content/docs/introducing-jsx.md [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
 
-### Why JSX? {#why-jsx}
+### Bakit JSX? {#bakit-jsx}
 
-React embraces the fact that rendering logic is inherently coupled with other UI logic: how events are handled, how the state changes over time, and how the data is prepared for display.
+Kaakibat ng pagre-render ang iba pang mga ginagawa sa UI--kinakatawan ng React ang ganitong ideya. Halimbawa nito ang paghahandle sa mga event, ang mga pabago-bago sa state, at ang paghahanda sa pagpapakita ng mga datos.
```
Comment (warrenca) [inline content/docs/introducing-jsx.md:23]: 👍 Kaakibat
UI to UI logic?

File: content/docs/introducing-jsx.md (hunk 2 above)
Comment (TheoryOfNekomata) [reply content/docs/introducing-jsx.md:17]: I don't want to translate word-per-word tho, so I guess Tagalog could use its own character regarding this.

File: content/docs/introducing-jsx.md (hunk 4 above)
Comment (TheoryOfNekomata) [reply content/docs/introducing-jsx.md:23]: The idea is still intact, I believe.

**Review by elrmndo — APPROVED** [review]

**Pull request merged**